rshim.spec.in

```diff
@@ -66,6 +68,24 @@ make
 %endif
 
 %post
+if [ -f /etc/os-release ]; then
+    . /etc/os-release
+    RHEL_MAJOR=$(echo $VERSION_ID | cut -d '.' -f 1)
+    RHEL_MINOR=$(echo $VERSION_ID | cut -d '.' -f 2)
+    if [ "$RHEL_MAJOR" -eq 9 ] && [ "$RHEL_MINOR" -eq 5 ]; then
+      polver=$(head -1 %{_datadir}/selinux/packages/rshim-fix.te | sed 's/;//' | awk '{print $3}')
+      echo "Updating SELinux policy to $polver for RHEL 9.5..."
+      if [ -x /usr/bin/checkmodule ] && [ -x /usr/bin/semodule_package ]; then
+        mkdir -p /tmp/rshim-selinux
+        checkmodule -M -m -o /tmp/rshim-selinux/rshim-fix.mod %{_datadir}/selinux/packages/rshim-fix.te
+        semodule_package -o /tmp/rshim-selinux/rshim-fix.pp -m /tmp/rshim-selinux/rshim-fix.mod
+        semodule -i /tmp/rshim-selinux/rshim-fix.pp || :
```

Comment:
This kind of change seems dangerous. It's better to get official approval from security ARCH.

Reply:
Another question: how will this change be removed when uninstalling the rshim driver?

Reply:
> This kind of change seems dangerous. It's better to get official approval from security ARCH.

I've tried to invite Ahmad Atamli <[ahmada@nvidia.com](mailto:ahmada@nvidia.com)> and Ahmad Atamli <[ahmada@nvidia.com](mailto:ahmada@nvidia.com)> for this code review, but their names are not shown in GitHub. Will send the patch over.

Reply:
> Another question: how will this change be removed when uninstalling the rshim driver?

Good question. We can leave it there as this is a SELinux fix that should do no harm to the system. But it may be a better idea to clean it up during uninstalling. 

Pushed the fix.